Update `Bancho` (#8094)
ppy/osu-wiki: wiki/Bancho_(server)/en.md

---
tags:
  - infrastructure
---

# Bancho (server)

*For other uses, see [Bancho (disambiguation)](/wiki/Disambiguation/Bancho).*

> **Bancho** (previously *osu!bancho*) is a code name of the server component which handles interactions between players running stable versions of osu!:

Reviewer: Weird, I don't remember renaming it but I guess I did in the client in a hope to stop thinking it was what we called the "official server" 🙄

Reviewer: ```suggestion
**Bancho** (previously *osu!bancho*) is a code name of the server component which handles real-time interactions between players running stable versions of osu!:
```

Author: I rechecked and it's called `osu!Bancho` in the chat welcome message, so you didn't, correct

- [Online presence](/wiki/Client/Interface/Chat_console#extended-chat-console) and the [IRC gateway for chat](/wiki/Community/Internet_Relay_Chat)
- [Spectator](/wiki/Gameplay/Spectating) mode

> - [Multiplayer system](/wiki/Client/Interface/Multiplayer) and [tournament management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)

Reviewer: Probably mention chat? Not just the "IRC gateway". 
It's the number one primary purpose of bancho after all.

Author: added to the first item

> During maintenance periods, Bancho is sometimes restarted. This is relayed to all online users via an in-game announcement system, as well as periodic warning messages in the chat with countdown. Bancho restarts do not affect score submission, since it is handled by different components.[^score-upload-ref]

Reviewer: ```suggestion
During maintenance periods, Bancho is sometimes restarted. This is relayed to all online users via an in-game announcement system, as well as periodic warning messages in the chat with countdown. Bancho restarts only affect chat and multiplayer, as all other functionality is handled via different APIs. [^score-upload-ref]
```
